src/import/engine.rs

```diff
@@ -102,7 +102,6 @@ fn tune_dboptions_for_bulk_load(opts: &DbConfig) -> (DBOptions, CFOptions) {
     db_opts.enable_statistics(false);
     // Vector memtable doesn't support concurrent write.
     db_opts.allow_concurrent_memtable_write(false);
-    db_opts.set_use_direct_io_for_flush_and_compaction(true);
```

Comment:
This fails tests on tmpfs, and since it is not actually important, I just remove it here.